### ISSUE TITLE
can't build bundle using this plugin

### ISSUE THREAD

**Issue by yarigpopov** [stealjs/steal-sass#1]
hello!

i tried to make Hello World application according to [guides](http://stealjs.com/docs/StealJS.quick-start.html) on site. 
here is a [repo ](https://github.com/chilicoder/stealjs-hello-world)

Everything works even with `css` and `less`

but if i try to use `scss` it works in development mode, but fails to build a bundle (`grunt build`)

to reproduce, clone [the repo](https://github.com/chilicoder/stealjs-hello-world) and then do 

```
npm install 
grunt build
```

**Comment by meuschke** [comment]
Same issue for me. Works well in the browser but when I build it with `steal-tools` I get the following error.

```
Running "steal-export:dist" (steal-export) task
OPENING: package.json!npm
WARN: Could not find css/aboutus.scss! in node_modules. Ignoring.
WARN: Could not find views/init.ejs! in node_modules. Ignoring.
WARN: Could not find core/control in node_modules. Ignoring.
WARN: Could not find yd/media/css/yd-core.scss in node_modules. Ignoring.
WARN: Could not find @empty in node_modules. Ignoring.
OUTPUT: +cjs
Fatal error: Error loading "css/aboutus.scss!steal-sass@0.0.3#main" at file:/Users/toni/Repos/JigMagga/yd/jig/infocenter/aboutus/css/aboutus.scss
Error loading "css/aboutus.scss!steal-sass@0.0.3#main" from "aboutus@1.0.0#aboutus" at file:/Users/toni/Repos/JigMagga/yd/jig/infocenter/aboutus/aboutus.js
Worker is not defined
```

**Comment by matthewp** [comment]
Hey @meuschke and @chilicoder sorry for missing these messages. Build is now supported in 1.0.0.

**Comment by matthewp** [comment]
Closing as this fixed by #2

**Comment by matthewp** [comment]
Note that it works with steal-tools >= 0.12.x
